### ISSUE TITLE
Why did you decide to make a copy instead of a fork from DP3-T?

### ISSUE THREAD

**Issue by kkoenen** [DEUS-BV/covid19-backend#1]
Looks like the [DP3-T repo](https://github.com/DP-3T/dp3t-sdk-backend/network/members) is unaware of this fork. This is not forbidden, but is not best practice either. Is there a specific reason not to fork the project in the context of Github?

**Comment by ryanbnl** [comment]
I also find the whole discussion about "the app which we developed" a little dishonest given that you've basically only made a couple of cosmetic commits.

That's akin to forking Linux, removing the copyright notices and changing some wording here and there and then claiming to be the creator of an open source operating system.

I understand that that sort of thing is common on politics but you'll win no friends with the technical community acting like this.

**Comment by kkoenen** [comment]
I agree with you. Advice would be; Just make a fork, so the origin is clear and there is no misunderstanding. I think the chosen DP3T platform can be an excellent start to base a Dutch solution on. So I think Deus is on the right track here. The real work is not so much creating an app (or, in this case, translating an existing app). The real tough work lies in adapting GGD processes, creating the social awareness, mitigating risks, orchestrating the efforts.. in other words; implementing the way of working in a society. That's where Deus will need to prove its worth. The app is just a tool.

**Comment by wmeints** [comment]
This repo is already missing out on a set of improvements in the original repo. They're working on automated dependency checks, security checks and tests. You're missing out here big time.

**Comment by kkoenen** [comment]
> 
> 
> This repo is already missing out on a set of improvements in the original repo. They're working on automated dependency checks, security checks and tests. You're missing out here big time.

Yes! This indeed is one of the main concerns under this question!
